src/plugins/monitoring/plugin.xml

```diff
@@ -5,7 +5,7 @@
     <name>Monitoring Service</name>
     <description>Monitors conversations and statistics of the server.</description>
     <author>IgniteRealtime // Jive Software</author>
-    <version>1.6.0</version>
+    <version>1.6.1</version>
     <date>03/05/2018</date>
```

Comment:
Probably should change the date here.

Reply:
Updated the release date and included issue number in changelog.